File: Mobile MediaPlayer (Xamarin Android)/MediaPlayer/MediaPlayer/Core/Playlist.cs
using System.Collections.Generic;

namespace MediaPlayer.Core
{
    public class Playlist
    {
        public string Name { get; private set; }
        public List<string> SongPaths { get; private set; }

        public Playlist(string name)
        {
            Name = name;
            SongPaths = new List<string>();
        }

        public void AddSong(string path)
        {
            SongPaths.Add(path);
        }

        public void AddListSong(List<string> paths)
        {
            SongPaths.AddRange(paths);
        }

        public void RemoveSong(string path)
        {
            while (SongPaths.Remove(path)) { };
        }
    }
}

File: Mobile MediaPlayer (Java Android)/app/src/main/java/com/example/mediaplayer/Core/Playlist.java
package com.example.mediaplayer.Core;

import java.util.ArrayList;
import java.util.List;

public class Playlist {
    private String Name;
    private List<String> SongPaths;

    public Playlist(String name)
    {
        Name = name;
        SongPaths = new ArrayList<>();
    }

    public String GetName()
    {
        return Name;
    }

    public List<String> GetSongPaths()
    {
        return SongPaths;
    }

    public void AddSong(String path)
    {
        SongPaths.add(path);
    }

    public void AddListSong(List<String> paths)
    {
        SongPaths.addAll(paths);
    }

    public void RemoveSong(String path)
    {
        while (SongPaths.remove(path)) { }
    }
}
